.github/workflows/ci.yml

```diff
@@ -31,6 +31,22 @@ jobs:
             rust: nightly-x86_64-pc-windows-msvc
             os: windows
             flags: /EHsc
+          - name: C++11
+            rust: nightly
+            os: ubuntu
+            flags: -std=c++11
+          - name: C++14
+            rust: nightly
+            os: ubuntu
+            flags: -std=c++14
```

Comment:
Unless you add `flags` as a matrix dimension outside of `include`, this job "overwrites" the previous one, and is overwritten by the next one, ultimately leaving only the C++20 job.

```diff
         os: [ubuntu]
+        flags: ['']
         include:
```

See https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations.

This is why you see only C++20 running on the PR.

![Screenshot from 2025-02-09 08-50-14](https://github.com/user-attachments/assets/27889d65-5198-4e0a-8a51-1f9a0e76875d)

Reply:
Thanks for catching this.  Done.